misk/api/misk.api

```diff
@@ -586,18 +588,20 @@ public final class misk/client/HttpClientSSLConfig {
 public final class misk/client/HttpClientsConfig : wisp/config/Config {
 	public static final field Companion Lmisk/client/HttpClientsConfig$Companion;
 	public fun <init> ()V
-	public fun <init> (Ljava/util/LinkedHashMap;Ljava/util/Map;)V
-	public synthetic fun <init> (Ljava/util/LinkedHashMap;Ljava/util/Map;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (Ljava/util/LinkedHashMap;Ljava/util/Map;Z)V
+	public synthetic fun <init> (Ljava/util/LinkedHashMap;Ljava/util/Map;ZILkotlin/jvm/internal/DefaultConstructorMarker;)V
 	public final fun component1 ()Ljava/util/LinkedHashMap;
 	public final fun component2 ()Ljava/util/Map;
-	public final fun copy (Ljava/util/LinkedHashMap;Ljava/util/Map;)Lmisk/client/HttpClientsConfig;
-	public static synthetic fun copy$default (Lmisk/client/HttpClientsConfig;Ljava/util/LinkedHashMap;Ljava/util/Map;ILjava/lang/Object;)Lmisk/client/HttpClientsConfig;
+	public final fun component3 ()Z
+	public final fun copy (Ljava/util/LinkedHashMap;Ljava/util/Map;Z)Lmisk/client/HttpClientsConfig;
+	public static synthetic fun copy$default (Lmisk/client/HttpClientsConfig;Ljava/util/LinkedHashMap;Ljava/util/Map;ZILjava/lang/Object;)Lmisk/client/HttpClientsConfig;
```

Comment:
These changes and the one in `BackwardsCompatibleClientsConfig` should be OK since these are instantiated via YAML parsing rather than direct constructor calls